applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -190,11 +184,11 @@ config NRF_21540_MAIN_TX_POWER_20DBM
 
 endchoice
 
-config NRF_21540_PRI_ADV_DBM
+config NRF_21540_MAIN_DBM
```

Comment:
The TX power setting for advertising is in ble_core.c, which  BIS gateway and and advertisment from peripheral for CIS will get benifit.
The TX power setting for connection is in cis_gateway and also cis_headset.

Reply:
It can be hard to understand, so I added some more explanation in the VS HCI .h file.
I also created OCT-2522 where we can potentially change these TX power VS HCI commands to make them easier to understand.